Sources/Helpers/OFFUrlsHelper.swift

```diff
@@ -20,6 +20,9 @@ class OFFUrlsHelper: NSObject {
     static func url(for country: Country) -> URL {
         return URL(string: "\(baseUrl)/country/\(country.code)")!
     }
+    static func urlCountry(for string: String) -> URL {
```

Comment:
we should keep the swift naming convention we adopted for other methods : `url(forCountry country: String)` and `url(forAllergen allergen: String)`

Reply:
Also, is this code related to the PR ?

Reply:
No it isn't. Was an experiment.

Reply:
I removed it